app/controllers/posts_controller.rb

```diff
@@ -18,4 +18,28 @@ def show
     @users = User.all
     @likes = Like.all
   end
+
+  def new
+    @post = Post.new
+  end
```

Comment:
- [ ] Kindly make sure that your inputs have some labels so that the user should know which field he/she is filling.
![image](https://user-images.githubusercontent.com/65068771/154373185-ab83d196-e9b1-4fe0-bbe5-b65bf25071cf.png)